app/less/.csslintrc

```diff
@@ -0,0 +1,3 @@
+{
+
```

Comment:
I don't mind using bootstrap's options here as long as it doesn't conflict with our [styleguide](https://github.com/thisissoon/angular-start/blob/master/app/less/README.md)